projects/plugins/social/src/class-meta-tags.php

```diff
@@ -0,0 +1,223 @@
+		'wp-caregiver/wp-caregiver.php',                         // WP Caregiver.
+		'wp-facebook-like-send-open-graph-meta/wp-facebook-like-send-open-graph-meta.php', // WP Facebook Like Send & Open Graph Meta.
+		'wp-facebook-open-graph-protocol/wp-facebook-ogp.php',   // WP Facebook Open Graph protocol.
+		'wp-ogp/wp-ogp.php',                                     // WP-OGP.
+		'zoltonorg-social-plugin/zosp.php',                      // Zolton.org Social Plugin.
+		'wp-fb-share-like-button/wp_fb_share-like_widget.php',   // WP Facebook Like Button.
+		'open-graph-metabox/open-graph-metabox.php',             // Open Graph Metabox.
+		'seo-by-rank-math/rank-math.php',                        // Rank Math.
+		'slim-seo/slim-seo.php',                                 // Slim SEO.
+	);
+
+	/**
+	 * This list is copied verbatim from class.jetpack.php
+	 *
+	 * @var array Plugins that conflict with Twitter cards.
+	 */
+	private $twitter_cards_conflicting_plugins = array(
+		'eewee-twitter-card/index.php',              // Eewee Twitter Card.
+		'ig-twitter-cards/ig-twitter-cards.php',     // IG:Twitter Cards.
+		'jm-twitter-cards/jm-twitter-cards.php',     // JM Twitter Cards.
+		'kevinjohn-gallagher-pure-web-brilliants-social-graph-twitter-cards-extention/kevinjohn_gallagher___social_graph_twitter_output.php',  // Pure Web Brilliant's Social Graph Twitter Cards Extension.
+		'twitter-cards/twitter-cards.php',           // Twitter Cards.
+		'twitter-cards-meta/twitter-cards-meta.php', // Twitter Cards Meta.
+		'wp-to-twitter/wp-to-twitter.php',           // WP to Twitter.
+		'wp-twitter-cards/twitter_cards.php',        // WP Twitter Cards.
+		'seo-by-rank-math/rank-math.php',            // Rank Math.
+		'slim-seo/slim-seo.php',                     // Slim SEO.
+	);
+
+	/**
+	 * Get a list of all active plugins.
+	 *
+	 * @return array Array of active plugins.
+	 */
+	public function get_active_plugins() {
+		$active_plugins = (array) get_option( 'active_plugins', array() );
+
+		if ( is_multisite() ) {
+			// Due to legacy code, active_sitewide_plugins stores them in the keys,
+			// whereas active_plugins stores them in the values.
+			$network_plugins = array_keys( get_site_option( 'active_sitewide_plugins', array() ) );
+			if ( $network_plugins ) {
+				$active_plugins = array_merge( $active_plugins, $network_plugins );
+			}
+		}
+
+		sort( $active_plugins );
+
+		return array_unique( $active_plugins );
+	}
+
+	/**
+	 * Check if meta tags should be rendered.
+	 *
+	 * @return bool True if meta tags should be rendered.
+	 */
+	public function should_render_meta_tags() {
+		if ( ! empty( array_intersect( $this->get_active_plugins(), $this->open_graph_conflicting_plugins ) ) ) {
+			return false;
+		}
+
+		if ( ! is_singular() ) {
+			return false;
+		}
+
+		if ( ! apply_filters( 'jetpack_enable_open_graph', true ) ) {
+			return false;
+		}
+
+		return true;
```

Comment:
I think we could consolidate these checks to:

```suggestion
		return apply_filters( 'jetpack_enable_open_graph', is_singular() ) );
```

Reply:
Good call, that's a nice simplification 👍🏼